src/index.tsx

```diff
@@ -1,13 +1,2 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
-import "./index.css";
-import App from "./App";
-
-const root = ReactDOM.createRoot(
-  document.getElementById("root") as HTMLElement
-);
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+import ImageGridComponent from "./components/ImageGrid/ImageGridComponent";
```

Comment:
1. change your current 2 lines to this one-liner
```suggestion
export * from "./components/ImageGrid/ImageGridComponent"
```

2. file extension here should be `index.ts` (no jsx/tsx inside)